src/Serialization/Fields.jl

```diff
@@ -510,10 +513,11 @@ function load_internal_with_parent(s::DeserializerState,
                                    dict::Dict,
                                    parent::FlintPadicField)
     rational_rep = load_type_dispatch(s, fmpq, dict[:rational_rep])
-    parent_field = load_type_dispatch(s, FlintPadicField, dict[:parent])
+    elem_precision = load_type_dispatch(s, Int64, dict[:precision])
 
-    # padic num precision is 1 higher than the field it lies in
-    if precision(parent_field) > precision(parent)
+    # this test should be moved in future ideally higher in the tree and possibly
+    # for all types that have precision
+    if elem_precision + 1 > precision(parent)
         @warn("Precision Warning: given parent is less precise than serialized parent",
```

Comment:
Is this warning even accurate, though? In `elem_precision` we stored the precision of the original value `n`, which may already back then have been different than the precision of its parent. The precision in the parent is only the *default* precision of a q-adic, AFAIK

Reply:
yes, the element can actually be more precise than it's parent. So we never actually lose information from the file.
So this warning doesn't do anything. Also, I don't need to throw this warning since I originally thought we may lose precision of the polynomial in the file if we load with a parent that is less precise, but this is not the case so I'll just remove this warning